src/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -14,13 +15,25 @@ export interface VideoPlayerProps {
    */
   mimeType?: string;
   loop?: boolean;
+  /**
+   * Video viewbox. If Different from container, video will be cropped
+   */
+  viewBox?: {
+    height?: string;
+    width?: string;
+  };
```

Comment:
Questa non va passata da fuori, è fissa sui vari breakpoint.

- Desktop: altezza 500px
- Tablet: altezza 400px
- Mobile: altezza 450px

La `width` viene impostata in modo da andare in "cover" sullo spazio disponibile